iogt_users/tests.py

```diff
@@ -3,19 +3,19 @@
 from wagtail.core.models import Site
 
 from home.factories import SiteSettingsFactory, HomePageFactory
-from iogt_users.factories import UserFactory
-from wagtail_factories import SiteFactory
+from iogt_users.factories import UserFactory, AdminUserFactory, GroupFactory
+from wagtail_factories import SiteFactory, PageFactory
 from questionnaires.factories import SurveyFactory
+from django.contrib.auth.models import Permission
 
 
 class PostRegistrationRedirectTests(TestCase):
     def setUp(self):
         self.user = UserFactory(has_filled_registration_survey=False)
-        self.admin_user = UserFactory()
 
-        Site.objects.all().delete()
-        self.site = SiteFactory(site_name='IoGT', port=8000, is_default_site=True)
-        self.home_page = HomePageFactory(parent=self.site.root_page)
+        root_page = PageFactory(parent=None)
+        self.home_page = HomePageFactory(parent=root_page)
+        self.site = SiteFactory(hostname='testserver', port=80, root_page=self.home_page)
 
         self.registration_survey = SurveyFactory(parent=self.home_page)
         self.site_settings = SiteSettingsFactory.create(
```

Comment:
`self.site_settings` is not used anywhere

Reply:
Fixed it!